src/components/OrphanTableRows.tsx

```diff
@@ -4,6 +4,7 @@ import * as Renderers from "../utils/Renderers";
 import * as StateUpdaters from "../utils/StateUpdaters";
 import * as Types from "./../utils/Types";
 
+
```

Comment:
There are some new lines in some files, if that's not necessary, we should remove those new lines.